### DISCUSSION TITLE
From September 28th, xDrip+ doesn't work with CareLink

### DISCUSSION THREAD

**Discussion by awbunge** [NightscoutFoundation/xDrip#3107] [Bug reports]
My phone: Samsung Galaxy S21 5G
Operative sistem: Android 13
xDrip+ Nightly build 24th September
From september 28th, when Medtronic, here in Rome (Italy) did an "Important scheduled maintenance work" xDrip+ don't work.
It's always the same "Login error".
Have some else the same problem?
What can we do?
Thanks.

**Comment by Navid200** [comment]
https://github.com/NightscoutFoundation/xDrip/discussions/3105
